### ISSUE TITLE
Disabled menu can't be seen completely in dark theme

### ISSUE THREAD

**Issue by azzamsa** [Alexhuszagh/BreezeStyleSheets#11]
Hi

I like how light theme behave. The disabled menu/button still appear and readable but can't be selected/highlighted. In contrary, those menu/button in dark theme completely unreadable

![company](https://user-images.githubusercontent.com/17734314/53292327-6e61e200-37f3-11e9-826d-a7d4ca06a7e4.png)

![company1](https://user-images.githubusercontent.com/17734314/53292330-73269600-37f3-11e9-8c37-a7af8af8696e.png)

is this expected behavior ?

Thanks for Breeze

**Comment by Alexhuszagh** [comment]
This has been fixed as of the latest commit. See the following images for enhancements to disabled menus, menu items, etc. Sorry for the delay, I needed to take a break from open source work for a long while. The best solution seems to be:

1. Don't provide selection backgrounds for disabled menu items.
2. Improve the color scheme for them.

![2021-07-17 00_07_40-Sample single widget application](https://user-images.githubusercontent.com/9440903/126026529-1f912e98-719f-40c7-b8f5-8ff34fb1a075.png)
![2021-07-17 00_07_51-Sample single widget application](https://user-images.githubusercontent.com/9440903/126026530-bf84451b-441b-49e7-a2f1-c83d974bda27.png)
![2021-07-17 00_07_26-Sample single widget application](https://user-images.githubusercontent.com/9440903/126026528-af775c5a-9a9e-4a75-8fb0-75ebecf12f63.png)
![2021-07-17 00_07_59-Sample single widget application](https://user-images.githubusercontent.com/9440903/126026531-4db5f535-e62c-46f7-a8a7-4b10d592fac5.png)
